--- pnetcdf-1.12.0/acinclude.m4.orig	2019-09-29 20:31:48.000000000 +0200
+++ pnetcdf-1.12.0/acinclude.m4	2019-12-08 14:24:10.766689962 +0100
@@ -1108,7 +1108,7 @@
         end module conftest
     ])]
 )
-dnl ac_try='$F90 ${F90FLAGS} conftest.f90 ${F90LIBS}>&AS_MESSAGE_LOG_FD'
+dnl ac_try='$F90 ${F90FLAGS} conftest.f90 ${F90LIBS} >AS_MESSAGE_LOG_FD 2>&1'
 dnl AC_TRY_EVAL(ac_try)
 if test -f conftest.${FC_MODEXT} ; then
    ac_cv_prog_f90_uppercase_mod=no
@@ -1652,7 +1652,7 @@
 AC_DEFUN([UD_CHECK_MPICC_IS_XLC],[
     AC_CACHE_CHECK([if MPI C compiler is IBM XLC], [ac_cv_mpicc_is_XLC],
     [ac_cv_mpicc_is_XLC=no
-     ac_MPICC_VER=`$MPICC -qversion >& conftest.ver`
+     ac_MPICC_VER=`$MPICC -qversion >conftest.ver 2>&1`
      ac_MPICC_VENDOR=`head -c 6 conftest.ver`
      if test "x${ac_MPICC_VENDOR}" = "xIBM XL" ; then
         ac_cv_mpicc_is_XLC=yes
@@ -1799,7 +1799,7 @@
 AC_DEFUN([UD_CHECK_MPIF77_IS_PGF77],[
     AC_CACHE_CHECK([if MPI Fortran 77 compiler is PGI pgf77], [ac_cv_mpif77_is_PGF77],
     [ac_cv_mpif77_is_PGF77=no
-     eval $MPIF77 -V </dev/null >& conftest.ver
+     eval $MPIF77 -V </dev/null >conftest.ver 2>&1
      ac_MPIF77_VENDOR=`head -c 5 conftest.ver`
      if test "x${ac_MPIF77_VENDOR}" = xPGF77 ; then
         ac_cv_mpif77_is_PGF77=yes
